.grit/patterns/js/no_alert_confirm_prompt.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Remove `alert`, `confirm` and `prompt` statement
+---
+
+JavaScript’s alert, confirm, and prompt functions are widely considered to be obtrusive as UI elements and should be replaced by a more appropriate custom UI implementation. Furthermore, alert is often used while debugging code, which should be removed before deployment to production.
+
+tags: #fix
+
+```grit
+engine marzano(0.1)
+language js
+
+call_expression(function=$name) => . where {
+    or {
+        $name <: `alert`,
+        $name <: `confirm`,
+        $name <: `prompt`,
+    }
```

Comment:
Note the idiomatic way to do this would be:
```
$name <: or { `confirm`, `prompt`, `alert` }
```

Reply:
That might be useful. Thanks!